index.html

```diff
@@ -203,6 +203,7 @@ <h1>{{ { US: 'USA', Intl: 'International' }[chosen_domain] }}
 var csse = null,   // Cleaned data loaded from csse.
     chart = null,  // The main object, Chartist object.
     theapp = null; // Main application controller, Vue object.
+    legendStates = []; // Legend state is saved separately for each domain
```

Comment:
Instead of putting legendStates here, let's put it on the Vue object.   It doesn't  need to be attached to the URL - it can go here in the "data" section - then after construction it will be available as `theapp.legendStates` (or ofthen this.legendStates.) 
  https://github.com/davidbau/covid-19-chart/pull/8/commits/be0401ddcba5e0223b93933ac9019ca290771263#diff-eacf331f0ffc35d4b482f1d15a887d3bL577